test/activity_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+
+require './lib/activity'
+
+class ActivityTest < Minitest::Test
+  def setup
+    @activity = Activity.new("Brunch")
+  end
+
+  def test_it_exists
+    assert_instance_of Activity, @activity
+  end
+
+  def test_it_has_a_name
+    assert_equal "Brunch", @activity.name
+  end
+
+  def test_it_defaults_to_no_participants
+    assert_equal ({}), @activity.participants
+  end
+
+  def test_it_can_add_a_participant
+    @activity.add_participant("Maria", 20)
+    assert_equal ({ "Maria" => 20 }), @activity.participants
+  end
+
+  def test_it_can_calculate_total_cost
+    @activity.add_participant("Maria", 20)
+    @activity.add_participant("Luther", 40)
+    assert_equal 60, @activity.total_cost
+  end
+
+  def test_it_can_add_multiple_participants
```

Comment:
I would move this test up above the total_cost test, just for readability it would be nice to group the add_participants tests.